js-intro-homework/js-excercise-1/js-excercise-1.js

```diff
@@ -0,0 +1,17 @@
+// a)
+let contry = 'Macedonia';
+let city = 'Bitola';
+let population = '74,550';
+// b)
+console.log(`County ${contry}, city ${city}, population ${population}`);
+// c)
+let isCapital = city === 'Skopje';
+// d)
+console.log(isCapital);
+// e)
+let capitalCity = 'Skopje';
+let dateVisited = '15/11/2018';
```

Comment:
For example this variable is good candidate for a let variable because it was change later in the code on line 16